src/observe.js

```diff
@@ -1685,7 +1685,8 @@
 
   var expose = global;
 
-  if (typeof exports !== 'undefined') {
+  if (typeof exports !== 'undefined' && exports + '' === '[object Object]' &&
```

Comment:
also, it would be great to use the same pattern that other libraries do, and if possible comment where it came from. The previous code came from underscore.js: https://github.com/Polymer/observe-js/pull/67#discussion_r19439906